tests/test_health.py

```diff
@@ -56,6 +56,16 @@ def test_full_health_check(self, mock_er, mock_ddb, mock_es):
         mock_res = health.l2_health_checks()
         self.assertDictEqual(healthy_res, mock_res)
 
+    @testmode.standalone
+    def test_resource_fetch(self):
+
+        service_tags = {"Key": "service", "Values": ["dss"]}
+        resource_list = health.get_resource_by_tag(resource_string='dynamodb:table', tag_filter=service_tags)
+        ddb_tables = [x['ResourceARN'].split('/')[1] for x in resource_list['ResourceTagMappingList'] if
+                      os.environ.get('DSS_DEPLOYMENT_STAGE') in x['ResourceARN']]
+        print(ddb_tables)
+        self.assertGreater(len(ddb_tables), 0)
```

Comment:
This test may be more future proof using `s3` instead of `dynamodb`.

Reply:
Updated test to work with s3